tests/src/Unit/AuthenticatedClientTest.php

```diff
@@ -310,36 +310,9 @@ private function fetchTokenLogger(int $count)
         for ($tokens = 0; $tokens < $count; ++$tokens) {
             // Since our class instantiates the Lock and passes in the logger, we have to expect these method calls
             // if we want to assert the last method call in this loop.
-            $logger->expects($this->any())->method('info')
+            $logger->expects($this->any())->method('debug')
                 ->withConsecutive(['Successfully acquired the "{resource}" lock.'],
-                    ['Expiration defined for "{resource}" lock for "{ttl}" seconds.'],
-                    [$this->callback(function ($message) {
-                        try {
-                            $this->assertEquals(
-                                'Retrieved a new mpx token {token} for user {username} that expires on {date}.',
-                                $message
-                            );
-                        } catch (ExpectationFailedException) {
-                            return false;
-                        }
-
-                        return true;
-                    }), $this->callback(function ($context) {
-                        try {
-                            $this->assertArraySubset([
-                                'token' => 'TOKEN-VALUE',
-                                'username' => 'mpx/USER-NAME',
-                            ], $context);
-                            $this->assertMatchesRegularExpression(
-                                '!\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\+\d{4}!',
-                                $context['date']
-                            );
-                        } catch (ExpectationFailedException) {
-                            return false;
-                        }
-
-                        return true;
-                    })]);
+                    ['Expiration defined for "{resource}" lock for "{ttl}" seconds.']);
```

Comment:
As I understand it, these two log events were moved from level `info` to level `debug` in Symfony 6.

Additionally I see we are dropping the test for our own logging, which happens with `info` level. Should we keep that test?

My preferred course of action would be to move our logging to `debug` and then make a minor tweak in the test:

```suggest
- $logger->expects($this->any())->method('info')
+ $logger->expects($this->any())->method('debug')
```

Reply:
Sorry git is making this look more convoluted than it is. 
I did change the two messages that are now `debug` into a new `::method()` call, but then also simplified the remaining `info` call to drop the callback, since now the strings are being compared as-is.

Reply:
> simplified the remaining info call to drop the callback, since now the strings are being compared as-is

Are we losing test coverage with this change?